addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -234,6 +234,19 @@ function checkMergeTableIsCancelHelpers() {
     ];
 }
 
+function checkMergeTable4IsMergeHelpers() {
```

Comment:
what do you think about something like:
```js
function tablesAreLinked(parent, ...children)
```
you would be able to give it any number of tables